main.py

```diff
@@ -0,0 +1,195 @@
+from fastapi import FastAPI
+from pydantic import BaseModel, Field
+from datetime import datetime
+import databases
+import sqlalchemy
+from hashlib import sha256
+from typing import List
+import pandas as pd
+from fastapi.responses import HTMLResponse, JSONResponse
+
+
+
+
+
+
+DATABASE_URL = "sqlite:///store_database.db"
+database = databases.Database(DATABASE_URL)
+metadata = sqlalchemy.MetaData()
+
+
+users = sqlalchemy.Table('users', metadata,
+sqlalchemy.Column('id', sqlalchemy.Integer, primary_key=True),
+sqlalchemy.Column('name', sqlalchemy.String(50)),
+sqlalchemy.Column('surname', sqlalchemy.String(70)),
+sqlalchemy.Column('email', sqlalchemy.String(128)),
+sqlalchemy.Column('password', sqlalchemy.String(129)),)
+
+products = sqlalchemy.Table('products', metadata,
+sqlalchemy.Column('id', sqlalchemy.Integer, primary_key=True),
+sqlalchemy.Column('name_prod', sqlalchemy.String(30)),
+sqlalchemy.Column('discription', sqlalchemy.String(1000)),
+sqlalchemy.Column('price', sqlalchemy.Float),)
+
+orders = sqlalchemy.Table('orders', metadata,
+sqlalchemy.Column('id', sqlalchemy.Integer, primary_key=True),
+sqlalchemy.Column('user_id', sqlalchemy.Integer, sqlalchemy.ForeignKey('users.id')),
+sqlalchemy.Column('prod_id', sqlalchemy.Integer, sqlalchemy.ForeignKey('products.id')),
+sqlalchemy.Column('status_order', sqlalchemy.Boolean, default=True),
+sqlalchemy.Column('created_at', sqlalchemy.TIMESTAMP, default=datetime.utcnow),)
+
+
+
+engine = sqlalchemy.create_engine(DATABASE_URL, connect_args={"check_same_thread": False})
+metadata.create_all(engine)
+app = FastAPI()
+
+class User(BaseModel):
+    id: int 
+    name: str = Field(..., title='Name', max_length=50)
+    surname: str = Field(..., title='Surname', max_length=70)
+    email: str = Field(..., title='Email', max_length=128 ) 
+    password: str = Field(..., title='Password', min_length=5)
+   
+   
+    def __repr__(self):
+        return f'User({self.name}, {self.surname},{self.email})'
+
+    
+
+class UserIn(BaseModel):
+    name: str = Field(..., title='Name', max_length=50)
+    surname: str = Field(..., title='Surname', max_length=70)
+    email: str = Field(..., title='Email', max_length=128 ) 
+    password: str = Field(..., title='Password', min_length=7)
```

Comment:
Модели очень похожи, а значит можно создать одну из них, наследуясь от другой.
```suggestion
class UserIn(BaseModel):
    name: str = Field(..., title='Name', max_length=50)
    surname: str = Field(..., title='Surname', max_length=70)
    email: str = Field(..., title='Email', max_length=128 ) 
    password: str = Field(..., title='Password', min_length=7)
    

class User(UserIn):
    id: int    
   
    def __repr__(self):
        return f'User({self.name}, {self.surname},{self.email})'
```